gallery/src/gallery-components/GalleryTiles.tsx

```diff
@@ -87,7 +89,12 @@ export const GalleryExampleTile: FunctionComponent<GalleryExampleTileProps> =
           defaultCheckeredBackground
         } = props,
 
-        [checkeredBackground, toggleCheckeredBackground] = useToggle(defaultCheckeredBackground || false),
+        location = useLocation(),
+
+        overrideCheckeredBackground = location.pathname.includes('noCheckeredBackground'),
```

Comment:
This is still making it a path parameter.  See `NxAccordionPage` for how to make it a query parameter

Reply:
> This is still making it a path parameter. See `NxAccordionPage` for how to make it a query parameter

Oh I completely missed the query params part. Changed here: c46c0ff64b7d09eb24486ef35694e8dfe79a8613